host/xdc/integration_failover_test.go

```diff
@@ -109,13 +111,18 @@ func (s *integrationClustersTestSuite) SetupSuite() {
 	var clusterConfigs []*host.TestClusterConfig
 	s.Require().NoError(yaml.Unmarshal(confContent, &clusterConfigs))
 
-	c, err := host.NewCluster(clusterConfigs[0], s.logger.WithTags(tag.ClusterName(clusterName[0])))
-	s.Require().NoError(err)
-	s.cluster1 = c
-
-	c, err = host.NewCluster(clusterConfigs[1], s.logger.WithTags(tag.ClusterName(clusterName[1])))
-	s.Require().NoError(err)
-	s.cluster2 = c
+	/*
```

Comment:
@yux0 let me know if this looks good to you until we decide what to do with this test.